data/json/martialarts.json

```diff
@@ -511,6 +511,8 @@
       "shishkebab_off",
       "shishkebab_on",
       "spear_knife",
+      "spear_knife_superior",
+      "spike_spike",
```

Comment:
```suggestion
      "spear_spike",
```

Reply:
Damn spiked spikes? I was still using regular spikes on my deathmobile.

Reply:
It's spikes that split off into even more spikes of course ;)